app/presenters/v3/service_offering_presenter.rb

```diff
@@ -42,12 +42,13 @@ def service_offering
         end
 
         def shareable(metadata)
-          metadata.nil? ? false : metadata['shareable'] == true
+          metadata['shareable'] == true
```

Comment:
If `service_offering.metadata = "null"` then `JSON.parse(service_offering.metadata)` will be `nil`, and there will be an exception

Reply:
This should return an empty object rather than a nil so it should not be a problem anymore